klasses/migrations/0030_alter_bootcamp_readable_id.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.25 on 2024-05-23 08:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("klasses", "0029_bootcamp_readable_id"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="bootcamp",
+            name="readable_id",
+            field=models.CharField(
+                blank=True,
+                help_text="The unique string to identify this bootcamp. It can be of the form 'bootcamp-v1:TYPE+TOPIC-FORMAT'(example: bootcamp-v1:public+IE-f2f)",
+                max_length=255,
+                null=True,
+                unique=True,
+            ),
```

Comment:
Maybe I am missing something, but could you help me understand why this AlterField migration is needed?
The field seems the same as https://github.com/mitodl/bootcamp-ecommerce/blob/master/klasses/migrations/0029_bootcamp_readable_id.py as readable_id is not required field until we do the backfil.
```
   operations = [
        migrations.AddField(
            model_name="bootcamp",
            name="readable_id",
            field=models.CharField(
                blank=True,
                max_length=255,
                null=True,
                unique=True,
                help_text="The unique string to identify this bootcamp. It can be of the form "
                "'bootcamp-v1:TYPE+TOPIC-FORMAT' (example: bootcamp-v1:public+IE-f2f)",
            ),
        ),
    ]
``

Reply:
I originally noticed this when running `makemigrations` was generating new migration everytime on master when it shouldn't.

That's probably because of the reformatting that might have been applied when you merged the PR. The only difference between 0029 & 0030 is the help_text. One has extra `""` probably because of the line break during the formatting.

We might have two ways to solve this extra migration issue:

1. The new migration that I created in this PR because the gets generated everytime on master because the migrations are out of sync with models?
2. We reformat the `help_text` so that it matches the migration in 0029.